packages/ffe-file-upload/less/file-upload.less

```diff
@@ -0,0 +1,307 @@
+        }
+    }
+
+    &__file-item-delete-icon {
+        right: 0;
+        content: '';
+        display: inline-block;
+        position: absolute;
+        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 200 200'%3E%3Cpath fill='@{ffe-blue-azure-fill}' d='M18.77,0c-3.562,0-7.125,1.347-9.834,4.044l-4.429,4.439c-5.381,5.394-5.381,14.25,0,19.68l71.19,71.83-71.66,71.83c-5.381,5.394-5.381,14.25,0,19.68l4.429,4.439c5.381,5.394,14.22,5.394,19.64,0l71.66-71.83,71.65,71.83c5.381,5.394,14.23,5.394,19.65,0l4.429-4.439c5.381-5.394,5.381-14.25,0-19.68l-71.19-71.83,71.66-71.83c5.381-4.914,5.381-13.78,0-19.21l-4.429-4.423c-5.381-5.394-14.22-5.394-19.64,0l-72.13,71.34-71.17-71.83c-2.691-2.697-6.257-4.044-9.819-4.044z'/%3E%3C/svg%3E");
+        height: 14px;
+        width: 14px;
+    }
+
+    &__file-item-delete-button {
+        padding-right: 20px;
+        padding-top: 5px;
+        margin-left: auto;
+        background-color: transparent;
+        border: none;
+        line-height: 18px;
+        position: relative;
+        color: @ffe-blue-azure;
+        font-size: 14px;
+        grid-column: 3;
+        grid-row: 1;
+
+        @media (min-width: @breakpoint-md) {
+            margin-right: 20px;
+        }
+
+        &:active,
+        &:focus,
+        &:hover {
+            text-decoration: underline;
+            color: @ffe-blue-royal;
+
+            > .ffe-file-upload__file-item-delete-icon {
+                background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 200 200'%3E%3Cpath fill='@{ffe-blue-royal-fill}' d='M18.77,0c-3.562,0-7.125,1.347-9.834,4.044l-4.429,4.439c-5.381,5.394-5.381,14.25,0,19.68l71.19,71.83-71.66,71.83c-5.381,5.394-5.381,14.25,0,19.68l4.429,4.439c5.381,5.394,14.22,5.394,19.64,0l71.66-71.83,71.65,71.83c5.381,5.394,14.23,5.394,19.65,0l4.429-4.439c5.381-5.394,5.381-14.25,0-19.68l-71.19-71.83,71.66-71.83c5.381-4.914,5.381-13.78,0-19.21l-4.429-4.423c-5.381-5.394-14.22-5.394-19.64,0l-72.13,71.34-71.17-71.83c-2.691-2.697-6.257-4.044-9.819-4.044z'/%3E%3C/svg%3E");
+            }
+
+            ~ .ffe-file-upload__file-item-filename {
+                text-decoration: underline;
+            }
+
+            ~ .ffe-file-upload__file-item-error-info
+                > .ffe-file-upload__file-item-error-info-filename {
+                text-decoration: underline;
+            }
+
+            ~ .ffe-file-upload__file-item-stencil-info
+                > .ffe-file-upload__file-item-stencil-info-filename {
+                text-decoration: underline;
+            }
+        }
+    }
+
+    &__file-item-stencil {
+        display: grid;
+        grid-template-columns: 1fr;
+
+        &-info {
+            display: grid;
+            grid-column: 1;
+            grid-row: 1;
+            grid-template-columns: 1fr;
+            overflow: hidden;
+
+            &-background {
```

Comment:
The level of nesting is rather confusing and hard to follow. Could you change the nesting so that element names are not nested? (i.e. `&__file-item-stencil-info-background` instead of `&__file-item-stencil &-info &-background`)

Reply:
> The level of nesting is rather confusing and hard to follow. Could you change the nesting so that element names are not nested? (i.e. `&__file-item-stencil-info-background` instead of `&__file-item-stencil &-info &-background`)

Sure :) I'll give it another look!